CMakeLists.txt

```diff
@@ -15,16 +15,19 @@ FetchContent_Declare(cmake-extensions
 FetchContent_Declare(ios-cmake
     GIT_REPOSITORY https://github.com/leetal/ios-cmake.git
     GIT_TAG 04d91f6675dabb3c97df346a32f6184b0a7ef845)
+FetchContent_Declare(arcana
+    GIT_REPOSITORY https://github.com/microsoft/arcana.cpp.git
+    GIT_TAG f2757396e80bc4169f2ddb938ce25367a98ffdd0)
```

Comment:
What will happen if the arcana Git tag listed here is different than the one listed in JsRuntimeHost? Will CMake complain? Or will it check out one on top of another?

Reply:
CMake docs say the first one it sees wins.